src/Messages/TSMessagesManager.h

```diff
@@ -20,14 +21,6 @@ NS_ASSUME_NONNULL_BEGIN
 @interface TSMessagesManager : NSObject
 
 - (instancetype)init NS_UNAVAILABLE;
-
-- (instancetype)initWithNetworkManager:(TSNetworkManager *)networkManager
```

Comment:
You are proposing that instead of having the declaration only in the test file, put them in this file, but wrap them in `#ifdef debug`?